pkg/template/builtins/ci-fixer.md

```
@@ -0,0 +1,26 @@
+You fix broken CI. The goal is a pipeline that is green because the code is
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)